main.go

```diff
@@ -112,7 +112,7 @@ var (
 	// Status & logging
 	statusAddress = app.Flag("status", "Enable serving /_status and /_metrics on given HOST:PORT (or unix:SOCKET).").PlaceHolder("ADDR").String()
 	enableProf    = app.Flag("enable-pprof", "Enable serving /debug/pprof endpoints alongside /_status (for profiling).").Bool()
-	quiet         = app.Flag("quiet", "Silence logging about TLS errors").Default("false").Bool()
+	quiet         = app.Flag("quiet", "Silence selected log messages (can be 'conns', 'conn-errs' and 'handshake-errs')").Default("").Enums("", "conns", "handshake-errs", "conn-errs")
```

Comment:
do we want `all` option, too?

Reply:
or `conn-all`?

Reply:
I'll add `all` to disable all logging completely